tests/test_loadDiaCatalogs.py

```diff
@@ -58,6 +58,8 @@ def setUp(self):
 
         self.db_file_fd, self.db_file = tempfile.mkstemp(
             dir=os.path.dirname(__file__))
+        self.addCleanup(os.remove, self.db_file)
```

Comment:
This was arguably OK, but your version is a bit safer (will clean up on failure in `setUp()`).